manifests/l/LightBurnSoftware/LightBurn/1.2.01/LightBurnSoftware.LightBurn.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.5-1-22610-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LightBurnSoftware.LightBurn
+PackageVersion: 1.2.01
+PackageLocale: en-US
+Publisher: LightBurn Software
+PublisherUrl: https://LightBurnsoftware.com/
+PublisherSupportUrl: https://forum.LightBurnsoftware.com/
+# PrivacyUrl: 
+# Author: 
+PackageName: LightBurn
+PackageUrl: https://LightBurnsoftware.com/
+License: Proprietary
+# LicenseUrl: 
+Copyright: Created by LightBurn Software, LLC
+# CopyrightUrl: 
+ShortDescription: LightBurn is layout, editing, and control software for your laser cutter.
+# Description: 
+Moniker: LightBurn
+Tags:
+- engrave
+- LightBurn
+# Agreements: 
+ReleaseNotesUrl: https://LightBurnsoftware.com/blogs/news/LightBurn-1-1-01-cleanup-for-a-few-small-issues
```

Comment:
```suggestion
ReleaseNotesUrl: https://lightburnsoftware.com/blogs/news/lightburn-1-2-01-patch-update-for-a-few-issues
```

Reply:
I updated the releasenotes link.

Reply:
Sorry for ignoring your commit. I have then updated the link and updated the case sensitivity of the other links.